packages/mos-gateway/src/mosHandler.ts

```diff
@@ -97,7 +97,7 @@ export class MosHandler {
 
 		const peripheralDevice = await coreHandler.core.getPeripheralDevice()
 
-		this._settings = peripheralDevice.settings || {}
+		this._settings = peripheralDevice.settings as any
```

Comment:
Could you explain the reasoning for this change? Casting to `any` isn't exactly in the spirit of making stronger typings, I think

Reply:
It's because the core version of these settings
https://github.com/nrkno/sofie-core/blob/release47/packages/shared-lib/src/core/model/peripheralDevice.ts#L28-L32
has nothing in common with the mos-gateway version
https://github.com/nrkno/sofie-core/blob/release47/packages/mos-gateway/src/mosHandler.ts#L36-L51

I don't think the core type should know about all these properties, that is the point of the config manifest after all, so I think there is something larger wrong with how we handle config for these devices that causes us to have weird/mismatched typings. Too much to deal with in the scope of this PR